src/core/include/databases.h

```diff
@@ -35,45 +36,66 @@ class Layer {
 };
 
 // class with net and temperatures
-// TODO: Fix it in future to make handy
 class HeatmapStorage {
  public:
-	using Heatmaps   = std::vector<Heatmap>;
 	HeatmapStorage() = default;
-	HeatmapStorage(std::vector<float> x_steps_mv, std::vector<float> y_steps_mv,
-	               const std::vector<float>& temperature, Box3D design_borders);
+	HeatmapStorage(std::vector<float> x_steps, std::vector<float> y_steps,
+	               const std::vector<float>& temperature,
+	               float environment_temperature, Box3D representation_borders);
 	[[nodiscard]] Box3D representation_borders() const {
 		return representation_borders_;
 	}
-	[[nodiscard]] float MinStep() const;
+	[[nodiscard]] float environment_temperature() const {
+		return env_temp_;
+	}
+
+	[[nodiscard]] float x_size() const {
+		return representation_borders_.coordinates()[0].second;
+	}
+	[[nodiscard]] float y_size() const {
+		return representation_borders_.coordinates()[1].second;
+	}
+	[[nodiscard]] float MinStep() const { return min_step_; }
+	[[nodiscard]] float max_temp() const;
+	[[nodiscard]] float min_temp() const;
 	[[nodiscard]] const Heatmaps& heatmaps() const { return heatmaps_; }
-	[[nodiscard]] const std::vector<float>& x_steps() const { return x_steps_; }
-	[[nodiscard]] const std::vector<float>& y_steps() const { return y_steps_; }
+	[[nodiscard]] const Floats& x_coords() const { return x_coords_; }
+	[[nodiscard]] const Floats& y_coords() const { return y_coords_; }
+
  private:
-	size_t layers_count_ = 0;  // IST in T2D file
-	std::vector<float> x_steps_;
-	std::vector<float> y_steps_;
+	size_t layers_count_           = 0;  // IST in T2D file
+	float env_temp_ = 0.0f;
+	float min_step_;
```

Comment:
1. Shall we define bothe of values then? 
2. If we have to new the proper env temp maybe we should use by default value which can't be interpretatewd as proper one?